addons/mrp/tests/test_manual_consumption.py

```diff
@@ -207,6 +207,8 @@ def get_moves(mo):
 
         # Pick manual move
         move_manual.picked = True
+        # Can't change the quantity when the move is picked
+        move_auto.picked = False
```

Comment:
Same question here. The move auto is indeed picked but I don't see difference by reset `picked` on false